index.html

```diff
@@ -10639,7 +10639,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				<pre class="example highlight">&lt;button aria-braillelabel="****"&gt;
   &lt;img alt="4 stars" src="images/stars.jpg"&gt;
 &lt;/button&gt;</pre>
-				<p>In the previous example, a braille display may display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
+				<p>In the previous example, a braille display can display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
```

Comment:
```suggestion
				<p>In the previous example, a braille display would display "btn ****" in Braille rather than the verbose "btn gra 4 stars".</p>			</div>
```